C2CRIBuildDir/projects/C2C-RI/src/RI_Logging/src/org/fhwa/c2cri/logger/RILogging.java

```diff
@@ -210,6 +209,7 @@ public void stopLogging() {
         oRoot.removeAppender(riAppender);
         oRoot.removeAppender(riGUIAppender);
         log.removeAppender(riAppender);
+        log.removeAppender(oRoot.getAppenders().get("C2CRIOutput"));
```

Comment:
Is this to remove the appender added in the RITestEngine? Is this "C2CRIOutput" appender guaranteed to exist at this call?